### ISSUE TITLE
userguide links from readme broken

### ISSUE THREAD

**Issue by g-k** [ThreatResponse/aws_ir#62]
```
$ curl -I http://aws-ir.readthedocs.io/en/latest/user_guide.html                                                                                              
HTTP/1.1 404 Not Found
```

Should they point to quickstart instead?

**Comment by andrewkrug** [comment]
@g-k Thanks for the heads up.  You are correct that we had to drop the link to the userguide in favor of the quick start.  I'd still like a userguide.  We just had so many new features I couldn't finish it before Black Hat '17.  

Do you want to PR the fix in?

**Comment by andrewkrug** [comment]
Closed by #63
